### PR TITLE
[ci-op-configs-mirror] Update only integration imagestreams to private ones

### PR DESCRIPTION
**Pull request by droslean**
/cc @petr-muller 

This PR adds an extra check to update only integration imagestreams instead. Also, update the `base_rpm_images` as well.

### CODE REVIEW

File: cmd/ci-op-configs-mirror/main.go [hunk 1]
```diff
@@ -87,6 +88,10 @@ func main() {
 			privateBaseImages(rbc.BaseImages)
 		}
 
+		if len(rbc.BaseRPMImages) > 0 {
+			privateBaseImages(rbc.BaseRPMImages)
```
Comment (stevekuznetsov) [inline cmd/ci-op-configs-mirror/main.go]: nit: functions with side-effects are hard to follow and we should prefer not to use them, instead using functions with clear data flow

File: cmd/ci-op-configs-mirror/main.go [hunk 2]
```diff
@@ -144,3 +149,10 @@ func privatePromotionConfiguration(promotion *api.PromotionConfiguration) {
 func strP(str string) *string {
 	return &str
 }
+
+func isIntegrationImageStream(name string) bool {
```
Comment (stevekuznetsov) [inline cmd/ci-op-configs-mirror/main.go]: Technically you need to walk every configuration we care about and see which image streams we promote to, right? Not everyone promotes to something prefixed with `4.`

Comment (petr-muller) [reply cmd/ci-op-configs-mirror/main.go]: It should be sufficient to use the same condition that is used in `promotesOfficialImages` check - we only mirror configs that promtoe into official imagestreams.

But in general yes, any reference to any IS that is a promotion target of the private org config should be changed.
